formula/nwchem.rb

```diff
@@ -0,0 +1,70 @@
+class Nwchem < Formula
+  desc "NWChem: Open Source High-Performance Computational Chemistry"
+  homepage "http://www.nwchem-sw.org"
+  url "https://github.com/nwchemgit/nwchem/releases/download/6.8.1-release/nwchem-6.8.1-release.revision-v6.8-133-ge032219-src.2018-06-14.tar.bz2"
+  version "6.8.1"
+  sha256 "23ce8241a5977a93d8224f66433851c81a08ad58a4c551858ae031485b095ab7"
+
+  depends_on "gcc" # for gfortran
+  depends_on "open-mpi"
+  depends_on "scalapack"
+  depends_on "openblas" => :recommended
+
+  def install
+    pkgshare.install "QA"
+
+    cd "src" do
+      (prefix/"etc").mkdir
+      (prefix/"etc/nwchemrc").write <<~EOS
+        nwchem_basis_library #{pkgshare}/libraries/
+        nwchem_nwpw_library #{pkgshare}/libraryps/
+        ffield amber
+        amber_1 #{pkgshare}/amber_s/
+        amber_2 #{pkgshare}/amber_q/
+        amber_3 #{pkgshare}/amber_x/
+        amber_4 #{pkgshare}/amber_u/
+        spce    #{pkgshare}/solvents/spce.rst
+        charmm_s #{pkgshare}/charmm_s/
+        charmm_x #{pkgshare}/charmm_x/
+      EOS
+
+      inreplace "util/util_nwchemrc.F", "/etc/nwchemrc", "#{etc}/nwchemrc"
+      inreplace "config/makefile.h" do |s|
```

Comment:
These options (`-mtune`, `-mfpmath`, and `-msse3`) should be left alone. Homebrew will handle them internally.